### PR TITLE
Add gemini samples

### PR DESCRIPTION
**Pull request by j-windsor**

### CODE REVIEW

File: python/gemini-api/main.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import google.generativeai as genai
```
Comment (msampathkumar) [inline python/gemini-api/main.py:3]: Does this import work?

```
import google.generativeai as genai
```

I am facing `ModuleNotFoundError: No module named 'google.generativeai'`

Comment (msampathkumar) [inline python/gemini-api/main.py:3]:
```suggestion
import os

import google.generativeai as genai
```

File: python/gemini-api/main.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+# Or use `os.getenv('GOOGLE_API_KEY')` to fetch an environment variable.
```
Comment (msampathkumar) [inline python/gemini-api/main.py]: Is something missing in below comment.

```
# Or use `os.getenv('GOOGLE_API_KEY')` to fetch an environment variable.
```

**Review by holtskinner — APPROVED** [review]

**Review by holtskinner — COMMENTED** [review]
General Question - Since this is a Cloud Repository, is there a reason it's using the Google AI (Non-Cloud) API for Gemini instead of Vertex AI?

**Review by msampathkumar — APPROVED** [review]
Python Samples are looking good.

**Comment by srithreepo** [comment]
> General Question - Since this is a Cloud Repository, is there a reason it's using the Google AI (Non-Cloud) API for Gemini instead of Vertex AI?

The Vertex AI Code samples will show up directly in the Code Sample explorer as soon as available on the Drift repo. The Google ones are not compatible with how Code sample browser is implemented so we had to add it here. In our next release the Getting started page will include a panel for Gen AI Development with the Vertex one in it as well.

**Review by muncus — APPROVED** [review]

**Review by pattishin — APPROVED** [review]

**Comment by jinhongy** [comment]
Members of our UX team have encountered some issues while following the instructions for the Python samples. Here are a few recommendations to improve the documentation:
- Some users may not have Python or the necessary dependencies installed. It would be helpful to provide links for quickly installing these dependencies, such as Google Cloud SDK and Python.
- Some individuals received errors when running the "pip" and "python" commands listed in the documentation. This was due to the Python version being used. It should be specified that "pip3" should be used instead of "pip" in these cases.
- The instructions could also include information on what to do after running the application locally to test the API. Providing additional instructions in this regard would be beneficial.

**Pull request merged**